.gitignore

```diff
@@ -0,0 +1 @@
+/.brackets.json
```

Comment:
Please remove this, since it will only apply to people who use this specific editor.

Note that you can add local-only ignore rules to the `.git/info/exclude` file within a given repository, so that will probably be helpful :)

Reply:
I removed .gitignore.